cmake/scipp-config.cmake.in

```diff
@@ -9,5 +9,5 @@ include("${CMAKE_CURRENT_LIST_DIR}/scipp-targets.cmake")
 check_required_components(scipp-targets)
 
 include(CMakeFindDependencyMacro)
-find_dependency(Boost 1.67)
 find_dependency(TBB)
+find_dependency(Boost 1.67)
```

Comment:
Accident, since I first tried to remove it (exporting boost headers into our package), but later changed to install boost via conda, so I re-added it.

Reply:
Do we still need the minimum requirement of 1.67? I thought that was to do with the units which we have now removed?

Reply:
Probably, but I don't see a good reason for moving to something older?